<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administrator</class>
 <widget class="QDialog" name="administrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/images/settings.png</normaloff>:/new/images/settings.png</iconset>
  </property>
  <widget class="QGroupBox" name="operatormodifygroupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>80</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator Modify</string>
   </property>
   <widget class="QLineEdit" name="operatorusername">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="operatorusernamelabel">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Operator Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="resetpasswordgroup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>225</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Reset Operator Password</string>
    </property>
    <widget class="QLineEdit" name="newpassword">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="resetbutton">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>29</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="deletegroup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>50</y>
      <width>225</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete Operator</string>
    </property>
    <widget class="QPushButton" name="deletebutton">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>26</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(252, 9, 0);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/images/delete.png</normaloff>:/new/images/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="okbutton">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>19</y>
      <width>65</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opusercancelbutton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>19</y>
      <width>65</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="settingsgroup">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>225</y>
     <width>481</width>
     <height>328</height>
    </rect>
   </property>
   <property name="title">
    <string>Application Settings</string>
   </property>
   <widget class="QGroupBox" name="admingroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>166</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Admin Password Change</string>
    </property>
    <widget class="QLabel" name="olpasswordlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Old Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="newpasswordlabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>New Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldpassedit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="newpassedit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="passokbutton">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>39</y>
       <width>47</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changebutton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QPushButton" name="passcancelbutton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scanintervalgroup">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>166</y>
      <width>151</width>
      <height>118</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Interval</string>
    </property>
    <widget class="QRadioButton" name="radiobutton2secs">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>24</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2 Secs</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radiobutton3secs">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>54</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3 Secs</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radiobutton4secs">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>84</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4 Secs</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="headerfootergroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Header-Footer</string>
    </property>
    <widget class="QLineEdit" name="header">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>19</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
      </font>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="footer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>46</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
      </font>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Header</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>46</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Footer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancelbutton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>73</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savebutton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>73</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="deviceselectgroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>461</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>Device select</string>
    </property>
    <widget class="QComboBox" name="devicecombobox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Sansui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Accurate Scales</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="devicelabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Device Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="newoperatorgroup">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New Operator</string>
   </property>
   <widget class="QLineEdit" name="newusernameedit">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>23</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newpasscancelbutton">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>22</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="newpaawordlabel">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>23</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>New Operator Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newpassokbutton">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>22</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newusernameedit</tabstop>
  <tabstop>newpassokbutton</tabstop>
  <tabstop>newpasscancelbutton</tabstop>
  <tabstop>operatorusername</tabstop>
  <tabstop>okbutton</tabstop>
  <tabstop>opusercancelbutton</tabstop>
  <tabstop>newpassword</tabstop>
  <tabstop>resetbutton</tabstop>
  <tabstop>deletebutton</tabstop>
  <tabstop>devicecombobox</tabstop>
  <tabstop>header</tabstop>
  <tabstop>footer</tabstop>
  <tabstop>oldpassedit</tabstop>
  <tabstop>passokbutton</tabstop>
  <tabstop>newpassedit</tabstop>
  <tabstop>changebutton</tabstop>
  <tabstop>passcancelbutton</tabstop>
  <tabstop>radiobutton2secs</tabstop>
  <tabstop>radiobutton3secs</tabstop>
  <tabstop>radiobutton4secs</tabstop>
  <tabstop>savebutton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
